templates/settings/bank-payment-method-settings.php

```diff
@@ -37,65 +45,121 @@
         </div>
 
         <div class="dokan-w10">
-            <textarea name="settings[bank][bank_addr]" rows="5" class="dokan-form-control" placeholder="<?php echo  esc_attr( ! empty( $fields_placeholders['bank_addr']['placeholder'] ) ? $fields_placeholders['bank_addr']['placeholder'] : '' ); ?>" <?php echo empty( $required_fields['bank_addr'] ) ? '' : 'required'; ?>><?php echo esc_html( $bank_addr ); ?></textarea>
+            <textarea name="settings[bank][bank_addr]" rows="5" class="dokan-form-control" placeholder="<?php echo esc_attr( ! empty( $fields_placeholders['bank_addr']['placeholder'] ) ? $fields_placeholders['bank_addr']['placeholder'] : '' ); ?>" <?php echo empty( $required_fields['bank_addr'] ) ? '' : 'required'; ?>><?php echo esc_html( $bank_addr ); ?></textarea>
+            <span class="error-container">
+                <?php
+                // phpcs:ignore
+                if ( ! empty( $_POST['error_bank_addr'] ) ) {
+                    echo '<span class="required">' . __( 'This is required', 'dokan-lite' ) . '</span>';
+                }
+                ?>
+            </span>
         </div>
     </div>
 
     <div class="dokan-form-group">
         <div>
-            <label><?php echo  esc_attr( ! empty( $fields_placeholders['iban']['label'] ) ? $fields_placeholders['iban']['label'] : '' ); ?> </label>
+            <label><?php echo esc_attr( ! empty( $fields_placeholders['iban']['label'] ) ? $fields_placeholders['iban']['label'] : '' ); ?> </label>
         </div>
 
         <div class="dokan-w10">
-            <input name="settings[bank][iban]" value="<?php echo esc_attr( $iban ); ?>" class="dokan-form-control" placeholder="<?php echo  esc_attr( ! empty( $fields_placeholders['iban']['placeholder'] ) ? $fields_placeholders['iban']['placeholder'] : '' ); ?>" type="text" <?php echo empty( $required_fields['iban'] ) ? '' : 'required'; ?>>
+            <input name="settings[bank][iban]" value="<?php echo esc_attr( $iban ); ?>" class="dokan-form-control" placeholder="<?php echo esc_attr( ! empty( $fields_placeholders['iban']['placeholder'] ) ? $fields_placeholders['iban']['placeholder'] : '' ); ?>" type="text" <?php echo empty( $required_fields['iban'] ) ? '' : 'required'; ?>>
+            <span class="error-container">
+                <?php
+                // phpcs:ignore
+                if ( ! empty( $_POST['error_iban'] ) ) {
+                    echo '<span class="required">' . __( 'This is required', 'dokan-lite' ) . '</span>';
+                }
+                ?>
+            </span>
         </div>
     </div>
 
     <div class="dokan-form-group">
         <div>
-            <label><?php echo  esc_attr( ! empty( $fields_placeholders['swift']['label'] ) ? $fields_placeholders['swift']['label'] : '' ); ?> </label>
+            <label><?php echo esc_attr( ! empty( $fields_placeholders['swift']['label'] ) ? $fields_placeholders['swift']['label'] : '' ); ?> </label>
```

Comment:
```
<?php echo ! empty( $fields_placeholders['swift']['label'] ) ? esc_attr( $fields_placeholders['swift']['label'] ) : '' ; ?>
```
is much more readable. Can you please change on the lines you've edited for this PR?